packages/stylelint-plugin/test.less

```diff
@@ -1,14 +1,12 @@
 // This is a file you can use to play with the stylelint plugin
-
 @import "~@kaizen/design-tokens/less/color-vars";
-
 @import "~@kaizen/design-tokens/less/spacing-vars";
```

Comment:
This is interesting... are these imports going to change when we no longer use the `var` prefix everywhere?